modules/controller/rpc-controller.js

```diff
@@ -503,7 +512,21 @@ class RpcController {
                 });
                 try {
                     let response = await this.dataService.runQuery(req.body.query, req.query.type.toUpperCase());
-
+                    const quads = [];
+                    const N3Parser = new N3.Parser({format: 'N-Triples', baseIRI: 'http://schema.org/'});
```

Comment:
maybe we can move this parsing into dataService

Reply:
I was thinking the same